### ISSUE TITLE
Fix missed links in the gold standard file

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#263]
In the original file sent to expert annotators, some segments missed a link to the actual original page. Hence there seem to be some missing pages in the corpus, but @ninpnin knows more about this.

**Comment by ninpnin** [comment]
I think there are two separate issues here:

1. Page links are generally missing in the beginning of the 90s because there are no PDFs available (at least right now)
2. There seem to be some random links missing

**Comment by MansMeg** [comment]
Re: 1) Is it possible to list which protocols that are missing PDFs? Then we can send it to KB/RB to be fixed?

**Comment by ninpnin** [comment]
On betalab, there's nothing past the 80s. On https://www.riksdagen.se/sv/dokument-lagar/?doktyp=prot , there are no PDFs for the beginning of the 90s.

**Comment by MansMeg** [comment]
@Lottabrorsson Do you know if you have these protocols as PDFs somewhere?

**Comment by Lottabrorsson** [comment]
> 

@MansMeg Yes we have the protocols from the beginning of the 90s as PDFs.

**Comment by MansMeg** [comment]
Do you know if it is possible to put them here:
https://www.riksdagen.se/sv/dokument-lagar/?doktyp=prot
?

Or somewhere where we can point toward them?

**Comment by Lottabrorsson** [comment]
> Do you know if it is possible to put them here: https://www.riksdagen.se/sv/dokument-lagar/?doktyp=prot ?
> 
> Or somewhere where we can point toward them?

I have sent the question to Lars.
